src/proposal/base/ModuleManager.sol

```diff
@@ -64,7 +64,7 @@ abstract contract ModuleManager is
     }
 
     modifier moduleLimitNotExceeded() {
-        if (uint8(_modules.length) >= MAX_MODULE_AMOUNT) {
+        if (_modules.length >= MAX_MODULE_AMOUNT) {
```

Comment:
Not sure whether I need my next coffee or not, but is it really "exceeding" the limit if `length == max_amount`? Shouldn't this be a `>` instead of `>=`? Note: I might be dumb, so feel free to ignore :D

Reply:
I would agree on that front
Should be > instead of >=

Reply:
I redact my approval xD
Because moduleLimitNotExceeded is a modifier that runs through before the new Module is added it has to include the max_module_amount to work properly

Reply:
Hm, okay, that makes slight sense to me, but still the math doesn't fully make sense then. Why isn't this then just changed from 

```
modifier moduleLimitNotExceeded() {
        if (_modules.length >= MAX_MODULE_AMOUNT) {
            revert Proposal__ModuleManager__ModuleAmountOverLimits();
        }
        _;
    }
```

to this

```
modifier moduleLimitNotExceeded() {
        _;
        if (_modules.length > MAX_MODULE_AMOUNT) {
            revert Proposal__ModuleManager__ModuleAmountOverLimits();
        }
    }
```

So the code runs first (`_;`) and then is being checked afterwards on whether it exceeded the actual limit?

Reply:
I know I'm a bit late to the party, but I agree with @marvinkruse  here. Should I open a new PR to tackle this or was it internally discussed to be correct?

@FHieser